sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ec2/files/manglefs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+to_delete=(
+  /etc/systemd/system/amazon-ssm-agent.service
+  /etc/systemd/system/multi-user.target.requires/coreos-metadata-sshkeys@core.service
+  /etc/amazon/ssm/
+  /etc/eks/bootstrap.sh
+)
+
+rm -rf "${to_delete[@]/#/${rootfs}}"
+
+cat > "${rootfs}/usr/lib/systemd/system/setup-oem.service" <<-'EOF'
+[Unit]
+Description=Setup OEM
+Before=amazon-ssm-agent.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/usr/bin/mkdir --parents /etc/amazon/ssm/ /etc/eks
+ExecStartPre=/usr/bin/ln --symbolic /usr/share/amazon/ssm/amazon-ssm-agent.json.template /etc/amazon/ssm/amazon-ssm-agent.json
+ExecStartPre=/usr/bin/ln --symbolic /usr/share/amazon/ssm/seelog.xml.template /etc/amazon/ssm/seelog.xml
+ExecStart=/usr/bin/ln --symbolic /usr/share/amazon/eks/bootstrap.sh /etc/eks/bootstrap.sh
```

Comment:
When the link already exists this will fail, do you want it to be skipped then? This would be possible with `ExecStartPre=-`.

Reply:
Since these files are likely touched by the user I would exclude them from the migration step. This means they will be there as regular files and not updated unless we could identify them as untouched with a checksum and create the symlink. For new instances the symlink is the default and we don't need to do a update logic and this is covered by the sysext content. The user could still overwrite the symlink or replace it with a file if we use `ExecStartPre=-`.